src/store.tsx

```diff
@@ -0,0 +1,7 @@
+import { createStore } from 'redux';
+import rootReducer from './reducers';
+
+export default createStore(
+  rootReducer,
+  window.__REDUX_DEVTOOLS_EXTENSION__ && window.__REDUX_DEVTOOLS_EXTENSION__()
```

Comment:
This one is for debug only. You can either remove it or put a comment here (e.g. // TODO: remove this line before deployment)

Reply:
Sorry, I forgot to remove it
Fixed